Ports/zlib/package.sh

```diff
@@ -2,14 +2,11 @@
 port=zlib
 version=1.2.11
 useconfigure=true
-files="https://www.zlib.net/zlib-${version}.tar.gz zlib-${version}.tar.gz
-https://www.zlib.net/zlib-${version}.tar.gz.asc zlib-${version}.tar.gz.asc"
-
-auth_type="sig"
-auth_import_key="783FCD8E58BCAFBA"
-auth_opts="zlib-${version}.tar.gz.asc"
+files="https://www.zlib.net/zlib-${version}.tar.gz zlib-${version}.tar.gz c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1"
+auth_type="sha256"
 
 configure() {
+    export CHOST=Linux
```

Comment:
Please add a comment to this explaining what's going on, like the commit message.

Reply:
Done!